backend/migrations/versions/799a2343955g_increase_quiz_question_title_length.py

```
@@ -0,0 +1,56 @@
+
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = "799a2343955g"
+down_revision = "799a2343955f"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Ruff (0.8.2)

Missing return type annotation for public function `upgrade`

Add return type annotation: `None`

(ANN201)